package.json

```diff
@@ -31,14 +31,14 @@
   },
   "homepage": "https://github.com/inuitcss/inuitcss#readme",
   "dependencies": {
-    "sass-mq": "^3.3.2"
+    "sass-mq": "^5.0.0"
   },
   "devDependencies": {
-    "glob": "^7.1.2",
-    "mocha": "^3.5.2",
-    "node-sass": "^4.5.3",
-    "sass-true": "^3.0.0",
-    "stylelint": "^8.1.1",
-    "stylelint-scss": "^2.1.0"
+    "glob": "^7.1.3",
+    "mocha": "^5.2.0",
+    "node-sass": "^4.9.3",
```

Comment:
`4.9.4` became available in the meantime 😉

Reply:
Done 😉